src/test/kotlin/org/rust/lang/core/completion/RsModCompletionTest.kt

```diff
@@ -0,0 +1,45 @@
+package org.rust.lang.core.completion
+
+class RsModCompletionTest : RsCompletionTestBase() {
+    fun testModCompletionSameDirectory() = checkSingleCompletionWithMultipleFiles("my_mod", """
+    //- main.rs
+    mod my_m/*caret*/;
+
+    fn main() {
+
+    }
+
+    //- my_mod.rs
+    pub fn test() {
+
+    }
+    """)
```

Comment:
Could you reformat this a bit? Indent the code between `//-` markers:

```
fun testFoo()  = check("""
//- main.rs
    bla-bla-bla
//- foo.rs
    bla-bla-bla
""")
```

Also, let's format empty functions as `fn main() {}` to save the line count.